src/rtext.c

```diff
@@ -647,7 +647,7 @@ GlyphInfo *LoadFontData(const unsigned char *fileData, int dataSize, int fontSiz
                     }
                 }
 
-                // Get bounding box for character (may be offset to account for chars that dip above or below the line)
+                // Get bounding box for character (maybe offset to account for chars that dip above or below the line)
```

Comment:
rtext.c line 650:  Is it safe to say "(will be offset ..." ?

Reply:
On seconds thoughts it should be used might... Wouldn't it fit better?

Reply:
both sound good to me